code/modules/antagonists/heretic/knowledge/blade_lore.dm

```diff
@@ -202,20 +206,18 @@
 	source.apply_status_effect(/datum/status_effect/protective_blades, 60 SECONDS, 1, 20, 0 SECONDS)
 
 /datum/heretic_knowledge/knowledge_ritual/blade
```

Comment:
Is there any reason for subtypes of knowledge ritual to exist anymore? The only reason they existed in the past was to handle construction of the tree, so can we nuke them now?

Reply:
mainly themeing, since the backgrounds would be the same if we removed the subtypes